Description: Patch systemd service to debian defaults
 ddupdate (0.6.1-1) sid; urgency=medium
 .
   * New upstream maintenance release
Author: Alec Leamas <leamas.alec@gmail.com>

--- ddupdate-0.6.1.orig/systemd/ddupdate.service
+++ ddupdate-0.6.1/systemd/ddupdate.service
@@ -5,7 +5,7 @@ After=network.target
 
 [Service]
 Type=oneshot
-ExecStart=/usr/local/bin/ddupdate
+ExecStart=/usr/bin/ddupdate
 Environment=PATH=/bin:/usr/bin:/sbin:/usr/sbin
 # User=ddupdate
 # Environment=http_proxy=my.proxy.domain:8888
